ansible-test/centos-stream8/build.sh

```diff
@@ -4,16 +4,21 @@
 set -ex
 
 SCRIPT_DIR=$(cd `dirname $0` && pwd -P)
-DEPENDENCIES="$(cat ${SCRIPT_DIR}/dependencies.txt | tr '\n' ' ')"
+DEPENDENCIES="$(cat ${SCRIPT_DIR}/dependencies.txt | grep -v '^#' | tr '\n' ' ')"
 
 build=$(buildah from quay.io/centos/centos:stream8)
 buildah run "${build}" -- /bin/bash -c "dnf update -y && dnf install --allowerasing -y ${DEPENDENCIES} && dnf clean all"
+# Set
 
 # Make sure en_US-UTF8 is around
 buildah run "${build}" -- /bin/bash -c "localedef --no-archive -i en_US -f UTF-8 en_US.UTF-8"
 
 # Extra python dependencies
-buildah run --volume ${SCRIPT_DIR}:/tmp/src:z "${build}" -- /bin/bash -c "pip3 install -r /tmp/src/requirements.txt"
+# python3.6 (default system interpreter)
+buildah run "${build}" -- /bin/bash -c "alternatives --set python3 /usr/bin/python3.6"
```

Comment:
Is this necessary? Shouldn't this already be the default one? Also even if the Python 3.8 (or later 3.9) install changes this, what's the result of this w.r.t. ansible-test? I don't think it should be needed.